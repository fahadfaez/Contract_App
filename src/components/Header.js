import React from 'react'
import { NavLink } from 'react-router-dom'

const Header =()=> (
    <header>
      <h1>Contract Database</h1>
      <NavLink to="/" activeClassName="is-active" exact={true}>Home</NavLink>
      <NavLink to="/create" activeClassName="is-active">Add Contract</NavLink>
      <NavLink to="/help" activeClassName="is-active">Help Pgae</NavLink>
    </header>
)

export default Header