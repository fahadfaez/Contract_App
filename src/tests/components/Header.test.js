import React from'react'
import ReactShallowRenderer from 'react-test-renderer/shallow'
import Header from '../../components/Header'



test('Should render Header Correctly',() => {

    const renderer = new ReactShallowRenderer()
    renderer.render(<Header />)
    expect(renderer.getRenderOutput()).toMatchSnapshot()
})