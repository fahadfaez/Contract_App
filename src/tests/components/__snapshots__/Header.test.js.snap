// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Header Correctly 1`] = `
<header>
  <h1>
    Contract Database
  </h1>
  <NavLink
    activeClassName="is-active"
    exact={true}
    to="/"
  >
    Home
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/create"
  >
    Add Contract
  </NavLink>
  <NavLink
    activeClassName="is-active"
    to="/help"
  >
    Help Pgae
  </NavLink>
</header>
`;
